src/libs/actions/BankAccounts.js

```diff
@@ -242,6 +242,47 @@ function validateBankAccount(bankAccountID, validateCode) {
     });
 }
 
+function openReimbursementAccountPage(stepToOpen, subStep, localCurrentStep) {
+    const onyxData = {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+                value: {
+                    errors: null,
+                    isLoading: true,
```

Comment:
is it this @ctkochan22 ?

Are we stuck in the isLoading state?

Reply:
but upon any sort of completion it should be set to false